README.md

```diff
@@ -52,183 +46,221 @@ Version|AppId
+#### System-wide installation
 
-### Season (weather) mods for ETS2MP/ATSMP
+Shell|System-wide search paths
+---|---
+bash|`$(pkg-config --variable=completionsdir bash-completion)` (e.g. `/usr/share/bash-completion/completions/`), `/usr/local/share/bash-completion/completions/`
+zsh|`/usr/share/zsh/site-functions/`, `/usr/local/share/zsh/site-functions/`
 
-Game|Proton|Wine
----|---|---
-ETS2|`$XDG_DATA_HOME/truckersmp-cli/Euro Truck Simulator 2/prefix/pfx/drive_c/users/steamuser/My Documents/ETS2MP/mod/`|`$XDG_DATA_HOME/truckersmp-cli/Euro Truck Simulator 2/prefix/pfx/drive_c/users/(os_login_name)/My Documents/ETS2MP/mod/`
-ATS|`$XDG_DATA_HOME/truckersmp-cli/American Truck Simulator/prefix/pfx/drive_c/users/steamuser/My Documents/ATSMP/mod/`|`$XDG_DATA_HOME/truckersmp-cli/American Truck Simulator/prefix/pfx/drive_c/users/(os_login_name)/My Documents/ATSMP/mod/`
+* The bash-completion file `truckersmp-cli.bash` needs to be renamed to `truckersmp-cli`
+* Debian-based systems are using the `/usr/share/zsh/vendor-completions/` directory for zsh completions
 
-See [TruckersMP Knowledge Base](https://truckersmp.com/knowledge-base).
+#### Per-user installation
 
-## Examples
+##### bash
 
-### Install Euro Truck Simulator 2
+Copy `truckersmp-cli.bash` to `$XDG_DATA_HOME/bash-completion/completions/truckersmp-cli`.
 
 ```
-$ ./truckersmp-cli -eu -n your_steam_account
+$ mkdir -p "${XDG_DATA_HOME:-~/.local/share}/bash-completion/completions"
+$ cp truckersmp-cli.bash "${XDG_DATA_HOME:-~/.local/share}/bash-completion/completions/truckersmp-cli"
 ```
 
-### Update Euro Truck Simulator 2 and start TruckersMP using Proton
+##### zsh
 
-```
-$ ./truckersmp-cli -eusp -n your_steam_account
-```
+Copy `_truckersmp-cli` to a directory that is part of `$fpath` and run `compinit`.
 
-### Only start TruckersMP without updating Euro Truck Simulator 2 using Wine
+## Runtime dependencies
 
-Note: Make sure Wine Steam is running in the same `$WINEPREFIX`!
+### Required
 
-```
-$ ./truckersmp-cli -esw
-```
+* [`python`][repology:python] in version 3.3 (released in September 2012) or later
+* `steam` either the [native Linux version][repology:steam] in use with Proton or the [Windows Steam][steam:windows] in use with Wine
+* [`sdl2`][repology:sdl2] as x86_64 version
 
-### Using a different prefix location
+### Optional
 
-Note:
-* While the prefix for Wine will point directly to the prefix location,
-Proton uses a subfolder `pfx` for the actual prefix and points to the parent folder.
-* Your prefix must be 64bits, the mod is not 32bits-compatible.
+* [`wine`][repology:wine] as a possible replacement to Proton
+* [`git`][repology:git] to clone this repo and help developing
+* [`vdf`][python:vdf] to automatically detect the steam account with saved credentials
```

Comment:
Sorry, I should have added the `pkg_resources` dependency here.
```markdown
* `pkg_resources` (part of [`setuptools`][python:setuptools]) to get version information from our Python package
```
Debian-based systems provide the `python3-pkg-resources` package.
This is not part of `python3-setuptools` package. `python3-setuptools` depends on `python3-pkg-resources`.

Reply:
This isn't even optional for installations as we don't have a version file there.

Reply:
I think it's optional for Python package users too. `pkg_resources` is used only when `--version` option is specified.

We already have `setuptools` in `setup.cfg`:
```ini
[options.extras_require]
optional = setuptools; vdf
```

Reply:
Hm, it's kinda hard to give support without knowing what version they're running. But we can try, I've set it as optional.